src/controllers/music/musicalbums.js

```diff
@@ -1,9 +1,20 @@
-define(['layoutManager', 'playbackManager', 'loading', 'events', 'libraryBrowser', 'imageLoader', 'alphaPicker', 'listView', 'cardBuilder', 'userSettings', 'globalize', 'emby-itemscontainer'], function (layoutManager, playbackManager, loading, events, libraryBrowser, imageLoader, AlphaPicker, listView, cardBuilder, userSettings, globalize) {
-    'use strict';
-
-    libraryBrowser = libraryBrowser.default || libraryBrowser;
-
-    return function (view, params, tabContent) {
+import playbackManager from 'playbackManager';
+import loading from 'loading';
+import events from 'events';
+import libraryBrowser from 'libraryBrowser';
+import imageLoader from 'imageLoader';
+import AlphaPicker from 'alphaPicker';
+import listView from 'listView';
+import cardBuilder from 'cardBuilder';
+import * as userSettings from 'userSettings';
+import globalize from 'globalize';
+import 'emby-itemscontainer';
+
+libraryBrowser = libraryBrowser.default || libraryBrowser;
```

Comment:
This hack isn't required in ES6. _Actually, it fails_
```suggestion

libraryBrowser = libraryBrowser.default || libraryBrowser;
```